tests/aws/services/lambda_/test_lambda_api.py

```diff
@@ -1636,6 +1636,55 @@ def test_notfound_and_invalid_routingconfigs(
             )
         snapshot.match("alias_does_not_exist_esc", e.value.response)
 
+    @markers.snapshot.skip_snapshot_verify(paths=["$..LoggingConfig"])
+    @markers.aws.validated
+    def test_alias_naming(self, aws_client, snapshot, create_lambda_function_aws, lambda_su_role):
+        """
+        numbers can be included and can even start the alias name, but it can't be purely a number
+        """
+        function_name = f"alias-fn-{short_uid()}"
+        create_response = create_lambda_function_aws(
+            FunctionName=function_name,
+            Handler="index.handler",
+            Code={
+                "ZipFile": create_lambda_archive(
+                    load_file(TEST_LAMBDA_PYTHON_ECHO), get_content=True
+                )
+            },
+            PackageType="Zip",
+            Role=lambda_su_role,
+            Runtime=Runtime.python3_9,
```

Comment:
Why python 3.9?

Reply:
copypasta

Reply:
Let's update them all when we do the LoggingConfig where we need to regenerate all the snapshots anyway

Reply:
we might be able to save a lot of time on image pulling (at least in some runners not testing the multi-runtime tests) if we can make our entire test suite use the new al2023 images (so much smaller: 3.11=>3.12 from 781.95 MB =>557.9 MB).